src/packages/excalidraw/CHANGELOG.MD

```diff
@@ -7,11 +7,14 @@ The change should be grouped under one of the below section and must contain PR
 - Chore: Changes for non src files example package.json.
 - Improvements: For any improvements.
 - Refactor: For any refactoring.
+
+Please add the latest features to the top!
 -->
 
 ## [Unreleased]
 
 ### Features
+- Add zoom to selection [#2522](https://github.com/excalidraw/excalidraw/pull/2522)
```

Comment:
```suggestion
- Add zoom to fit selection [#2522](https://github.com/excalidraw/excalidraw/pull/2522)
```

Reply:
Not that it matters, but `zoom to selection` is how all other apps call it.